R/ui.R

```diff
@@ -0,0 +1,165 @@
+#' @include globals.R
+
+# Mount OpenAPI and UI
+#' @noRd
+mountUI <- function(pr, host, port, ui_info, callback) {
+
+  # Build api url
+  api_url <- getOption(
+    "plumber.apiURL",
+    urlHost(
+      scheme = getOption("plumber.apiScheme", "http"),
+      host   = getOption("plumber.apiHost", host),
+      port   = getOption("plumber.apiPort", port),
+      path   = getOption("plumber.apiPath", ""),
+      changeHostLocation = TRUE
+    )
+  )
+
+  # Mount openAPI spec paths openapi.json
+  mountOpenAPI(pr, api_url)
+
+  # Mount UIs
+  ui_mount <- .globals$interfaces$mount[[ui_info$ui]]
+  if (!is.null(ui_mount)) {
+    ui_url <- do.call(ui_mount, c(list(pr, api_url), ui_info$args))
+    message("Running ", ui_info$ui, " UI at ", ui_url, sep = "")
+  } else {
+    message("Unknown user interface \"", ui_info$ui,"\". ",
+            ". Maybe try library(", ui_info$ui,").")
+    return(NULL)
```

Comment:
The return value is not used. For readability, do not return anything.

```suggestion
    return()
```

Reply:
across package it uses return(NULL) which is the similar to return() as I understand it. Should we use a consistent return across package?

Reply:
Correct. They have the same returned value.

> Should we use a consistent return?

I'm all for it. (Not a big priority)

If it's for the purposes of early returning, let's use `return()`.

If the value means something, `return(NULL)`.